packages/html2react/src/libraries/__tests__/component.tests.tsx

```diff
@@ -1,11 +1,20 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
The processors in the first test in the file are empty functions so had to add that to the file to please eslint.